lib/game.ex

```diff
@@ -124,4 +128,128 @@ defmodule Game do
   def get_marker(game), do: game.player_2_mark
 
   def update_board(board, game), do: %{game | board: board}
+
+  def next_open(game), do: List.first(get_available_moves(game))
+
+  def handle_next_move(game) do
+    move = next_open(game)
+    player_move_list = determine_move_list(game)
+    game = update_available_moves(move, game)
+    game = update_player_move_list(game, player_move_list, move)
+    place_marker(move, game)
+  end
+
+  def best_move(game) do
```

Comment:
This is the current iteration when I had an issue with trying to update the best_score outside of the map function.